install.sh

```diff
@@ -357,13 +357,19 @@ nvm_do_install() {
     # shellcheck disable=SC2016
     if ${BASH_OR_ZSH} && ! command grep -qc '$NVM_DIR/bash_completion' "$NVM_PROFILE"; then
       echo "=> Appending bash_completion source string to $NVM_PROFILE"
+      if [ -n "$ZSH_VERSION" ]; then
```

Comment:
also `${ZSH_VERSION-}` here

Reply:
Removed in favor of using `bash_completion`